test/provision/terraform/main.tf

```diff
@@ -29,8 +29,8 @@ variable "inventory_output" {
   default = "./inventory.ec2"
 }
 
-module "otel-env-provisioner" {
-  source             = "git::https://github.com/newrelic-experimental/otel-env-provisioner//terraform/otel-ec2"
+module "env-provisioner" {
+  source             = "git::https://github.com/newrelic-experimental/env-provisioner//terraform/otel-ec2"
```

Comment:
I guess the module alerts: https://github.com/newrelic/opentelemetry-collector-releases/blob/9d674974cefe0798ecc01ee9ff2d7fd9bcd9b385/test/canaries/alerts/main.tf#L3 should also be updated, isn't it?

Reply:
You're totally right!

Reply:
Great